File: DropZone/DropZone.Core/Models/User/UserModel.cs
using System;

namespace DropZone.Core.Models
{
    public class UserModel
    {
        public long Id { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string Name { get; set; }

        public string Surname { get; set; }

        public string Phone { get; set; }

        public string Address { get; set; }

        public DateTime Birthday { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long RoleId { get; set; }

        public long? DropZoneId { get; set; }


        public RoleModel Role { get; set; }

        public DropZoneModel DropZone { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/users/user.model.ts
import { RoleModel } from "./role.model";
import { DropZoneModel } from "../dropzones/dropzone.model";

export class UserModel {
	public Id: number;
	public Email: string;
	public Password: string;
	public Name: string;
	public Surname: string;
	public Phone: string;
	public Address: string;
	public Birthday: Date;
	public ModifiedAt: Date;

	public RoleId: number;
	public DropZoneId: number;
	
	public Role: RoleModel;
	public DropZone: DropZoneModel;
};